packages/core/lib/commands/test/run.js

```diff
@@ -107,6 +107,27 @@ module.exports = async function (options) {
     }
   };
 
+  function sanitizeNetworkID(network_id) {
```

Comment:
I would actually rather see this function be a bit more general, `sanitizeGanacheOptions`. In there we could handle `network_id`, I can foresee in the future a time that we might concern ourselves with sanitizing other options as well.

Reply:
Okay, I will do that.